src/main/java/Cart.java

```diff
@@ -0,0 +1,11 @@
+import java.util.List;
+
+public class Cart {
```

Comment:
What is the point for `Cart `to have `String itemName` when we have already defined `Inventory`?

In my opinion `Cart `class should contain:
-` Inventory item`
-` int qty`

Reply:
Incorporated